P2047-alloc-aware-optional.html

```diff
@@ -38,128 +38,70 @@
 <h1 align=center>An allocator-aware optional type</h1>
 
 <h2>Abstract</h2>
+ 
 
-
-<!--
 <h2>change history :</h2>
-<p> - modified alloc-optional(const alloc-optional&) to use default resource </p>
-<p> - added allocator extended value_or</p>
-<p> - swapped around in_place_t and allocator_arg_t positions in allocator extended in_place_t constructors </p>
-<p> - extended feedback section </p>
- -->
-<p>
-  Library types that can potentially hold <em>allocator-aware (AA)</em>
-  objects should, themselves, be allocator-aware. A PMR container, for example,
-  depends on AA types following a known AA protocol so that it (the container)
-  can uniformly manage its memory. Even types that don't manage their own
-  memory, such as <tt>tuple</tt>, follow the AA rules when they hold one more
-  more AA elements. (A special case is <tt>pair</tt>, which is not actually AA
-  but effectively follows the rules through special handling
-  in <em>uses-allocator construction</em>.)
-</p>
+- replaced <tt>alloc_optional</tt> with additional overloads of <tt>make_optional</tt> that 
```

Comment:
Suggestion: move the change history after the abstract. It's conventional that the abstract is and remains the very first thing in a paper; the change history can come after it just fine.

Reply:
done